test/integration/targets/freebsd_update/aliases

```diff
@@ -0,0 +1,3 @@
+skip/osx
+skip/rhel
+skip/docker
```

Comment:
To run the tests in CI you'll need to add another alias:

```
shippable/posix/group1
```

This test is also destructive and requires root privileges, so you'll also want to add:

```
destructive
needs/root
```